### PR TITLE
Add offline mode (Not ready to merge)

### PR DESCRIPTION
**Pull request by eroshenkoam**

### CODE REVIEW

**Comment by eroshenkoam** [comment]
test this please

**Comment by baev** [comment]
summon @lanwen

**Comment by lanwen** [comment]
:+1: after quick view

File: src/main/java/ru/yandex/qatools/allure/jenkins/AllureReportPlugin.java [hunk 1]
```diff
@@ -19,7 +20,7 @@
 
     /**
      * @deprecated since 1.0
-    */
+     */
     @Deprecated
     public static final String REPORT_PATH_OLD = "allure-reports";
```
Comment (baev) [inline src/main/java/ru/yandex/qatools/allure/jenkins/AllureReportPlugin.java]: maybe remove it?

**Comment by baev** [comment]
LGTM

**Comment by baev** [comment]
test this please

**Comment by baev** [comment]
test this please

**Pull request merged**